src/AngleSharp.Scripting.JavaScript/SystemTypeConverter.cs

```diff
@@ -48,9 +48,9 @@ public static Object ToObject(JsValue arg)
             return arg.ToObject();
         }
 
-        static Object Clr(this JsValue? arg)
+        static Object Clr(this JsValue arg)
         {
-            return arg.HasValue ? arg.Value.ToObject() : null;
+            return arg;
```

Comment:
Reasons for this? The Clr function converts the JsValue (wrapping a non-JS value) to its original (non-JS) value.